frontend/packages/dev-console/src/components/import/serverless/ServerlessSection.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as _ from 'lodash';
+import { stateToProps } from '@console/internal/reducers/features';
+// import { connectToFlags } from '@console/internal/reducers/features';
+import { FLAG_KNATIVE_SERVING } from '@console/knative-plugin';
+import FormSection from '../section/FormSection';
+import { CheckboxField } from '../../formik-fields';
+import './ServerlessSection.scss';
+
+type ServerlessSectionProps = {
+  flags: { [key: string]: boolean };
+};
+
+const ServerlessSection: React.FC<ServerlessSectionProps> = ({ flags }) => {
+  if (flags[FLAG_KNATIVE_SERVING]) {
+    return (
+      <FormSection title="Serverless Options" divider>
+        <label className="pf-c-button odc-serverless-tech-preview">Tech Preview</label>
+        <CheckboxField
+          type="checkbox"
+          label="Enable scaling to zero when idle"
+          name="serverless.trigger"
+        />
+      </FormSection>
+    );
+  }
+
+  return null;
+};
+
+// let FLAGS: ServerlessSectionProps ;
+
+// export default connectToFlags(FLAGS[FLAG_KNATIVE_SERVING])(ServerlessSection);
+
+export default connect(
+  (state) => stateToProps([FLAG_KNATIVE_SERVING], state),
+  null,
+  null,
+  { areStatePropsEqual: _.isEqual },
+)(ServerlessSection);
```

Comment:
@christianvogt You commented on the other PR that copies this one to change `stateToProps` to use `connectToFlags`. 
I tried it here but didn't get it to work. Please look at the comment out lines above. 
I would add that `connectToFlags` is just a wrapper around this exact set of lines. So there appears to real benefit.